pkg/api/types.go

```diff
@@ -1188,6 +1189,8 @@ func (p ClusterProfile) ClusterType() string {
 		return "aws-china"
 	case ClusterProfileAWSGovCloud:
 		return "aws-usgov"
+	case ClusterProfileAzure2:
+		return "azure-2"
```

Comment:
This likely belongs to the case below, this is not a differnt cluster type, just a new account for an existing cluster type

Reply:
I was planning to make the tests to use it as `azure-2` cluster profile. see https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L64

So that wasn't necessary, right?

Reply:
Yes, but cluster profile is something differnt than cluster type

Reply:
I don't see why we would have a differnt clsuter type for 2nd auzre account? We did not have different cluster type for 2nd gcp or aws accounts. This just creates a pitfall for step authors.

Reply:
Ok I will remove this, and its reference from the steps.

Reply:
https://github.com/openshift/release/pull/21745